src/proxyWorker.js

```diff
@@ -254,6 +254,28 @@ document.createElement = function document_createElement(what) {
         }
       });
 
+      var style = {
+        parentCanvas: canvas,
+        removeProperty: function(){},
+        setProperty:  function(){},
+      };
+
+      Object.defineProperty(style, 'cursor', {
+        set: function(value) {
+          if (!style.cursor_ || style.cursor_ !== value) {
+            style.cursor_ = value;
+            if (style.parentCanvas === Module['canvas']) {
+              postMessage({ target: 'canvas', op: 'style', property: 'cursor', cursor: style.cursor_ });
```

Comment:
how about sending `op: 'set', property: 'cursor', value: ...`, then the client side can just do `Module.canvas[data.property] = data.value`?